indigo_app/templates/indigo_api/work_overview.html

```diff
@@ -60,15 +60,16 @@ <h4 class="text-muted">{{ work.frbr_uri }}</h4>
                   <dt>Commencement</dt>
                   <dd>
                     {% if work.commenced %}
-                      Commenced on
-                      {% if work.commencement_date %}
-                        {{ work.commencement_date|date:"Y-m-d" }}
-                      {% else %}
-                        <i>(unknown)</i>
-                      {% endif %}
-                      {% if work.commencing_work %}
-                        by <a href="{% url 'work' frbr_uri=work.commencing_work.frbr_uri %}" data-popup-url="{% url 'work_popup' frbr_uri=work.commencing_work.frbr_uri %}">{{ work.commencing_work.title }}</a>
-                      {% endif %}
+                      {% with work.commencements.count as n_commencements %}
```

Comment:
`.count()` will go to the db, you can also use `work.commencements|length` which will test the length of the object, which will be evaluated once. Although it's moot because you use the variable, so this is just an FYI.